src/RawData/storageController.ts

```diff
@@ -67,24 +72,24 @@ export const readRawDataParameter = async (
 };
 
 export const processRawDataFile = async (fileId: string) => {
```

Comment:
This entire function is now very complicated. Can we please split it to reduce cognitive complexity? Maybe introduce some new `storageService` and `storageRepository` files and folders to take care of some of the different functionalities.

Reply:
Done